src/Microsoft.NetCore.Analyzers/Core/Security/DoNotUseDeprecatedSecurityProtocols.cs

```diff
@@ -193,6 +179,51 @@ public override void Initialize(AnalysisContext context)
                         },
                         OperationKind.SimpleAssignment,
                         OperationKind.CompoundAssignment);
+
+                    return;
```

Comment:
What is it used for?

Reply:
Just to show that lines afterwards aren't executing as part of the code block, since they're local functions.